FV3/conf/configure.fv3.linux_gnu

```diff
@@ -38,6 +38,10 @@ FFLAGS := $(INCLUDE) -fcray-pointer -ffree-line-length-none -fno-range-check -fP
 CPPDEFS += -Duse_libMPI -Duse_netCDF -DSPMD -DUSE_LOG_DIAG_FIELD_INFO -DUSE_GFSL63 -DGFS_PHYS 
 CPPDEFS += -DNEW_TAUCTMAX -DINTERNAL_FILE_NML -DNO_INLINE_POST
 
+ifeq ($(GT4PY_DEV),Y)
```

Comment:
This will be false.
Not that these changes are simply a propagation of how it is currently already dealt with in `configure.fv3.gnu_docker` in order to make the files consistent.